generators/app/templates/gulp/unit-tests.js

```diff
@@ -11,7 +11,9 @@ function runTests (singleRun, done) {
     configFile: path.join(__dirname, '/../karma.conf.js'),
     singleRun: singleRun,
     autoWatch: !singleRun
-  }, done);
+  }, function() {
```

Comment:
Here is the SO post re: this issue

http://stackoverflow.com/questions/26614738/issue-running-karma-task-from-gulp